playbooks/group_vars/webportals.yml

```diff
@@ -86,7 +86,7 @@ webportal_git_config:
   - name: user.name
     value: "SkynetLabs Devs"
   - name: user.email
-    value: "{{ skynetlabs_devs_email }}"
+    value: "{{ support_email }}"
```

Comment:
We should split one email var to `ssl_support_email` and `git_checkout_email` vars, because portal operators could use 2 different emails for this.